EventFilter/CTPPSRawToDigi/interface/CTPPSPixelDataFormatter.h

```diff
@@ -53,31 +55,36 @@ class CTPPSPixelDataFormatter {
   typedef std::map<int, FEDRawData> RawData;
   typedef std::vector<CTPPSPixelDigi> DetDigis;
 
+  typedef std::vector<CTPPSPixelDataError> DetErrors;
+  typedef std::map<uint32_t, DetErrors> Errors;
+
   typedef uint32_t Word32;
   typedef uint64_t Word64;
 
   CTPPSPixelDataFormatter(std::map<CTPPSPixelFramePosition, CTPPSPixelROCInfo> const &mapping);
 
+  void setErrorStatus(bool ErrorStatus);
+
   int nWords() const { return theWordCounter; }
 
-  void interpretRawData( bool& errorsInEvent, int fedId,  const FEDRawData & data, Collection & digis);
+  void interpretRawData( bool& errorsInEvent, int fedId,  const FEDRawData & data, Collection & digis, Errors & errors);
 
 
 
 private:
 
   mutable int theWordCounter;
 
+  bool includeErrors;
```

Comment:
please add some decoration for the new data member (this class has a mix of two styles already: "the" and "m_" prefix). It would have been nicer to have just one.

Reply:
done